subiquity/server/server.py

```diff
@@ -629,7 +629,10 @@ def use_passwd(passwd):
 
         self.installer_user_name = username
 
-        if self._user_has_password(username):
+        if username is None:
+            # extract_default can return None, if there is no default user
+            self.installer_user_passwd_kind = PasswordKind.NONE
+        elif self._user_has_password(username):
```

Comment:
Makes sense, so the error we are avoiding here is `_user_has_password` fails with something like:
```
TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'
```

Reply:
Yes, that's exactly what happens